index.ts

```diff
@@ -20,10 +20,14 @@ async function makeNoise() {
     const randomChoice = rnd(2)
     const userData = faker.helpers.contextualCard()
     
-    await posthog.capture('user signup', { ...userData, distinct_id: userData.username })
+    await posthog.capture('user signup', { ...userData, distinct_id: userData.username, $groups: {'organization': userData.company.name} })
```

Comment:
This needs to also groupidentify at some point - otherwise the group will have no properties. :)